ctx.go

```diff
@@ -370,10 +371,60 @@ func (c *DefaultCtx) Response() *fasthttp.Response {
 	return &c.fasthttp.Response
 }
 
+type Fmt struct {
```

Comment:
Please add some comments

Reply:
And not sure `Fmt` is good choice as name. Perhaps choose more explanatory name

Reply:
`ResFmt`?

Reply:
Yes this needs comments and a better name.  ResFmt seems better to me.  What do you think @efectn